cmd/action/verify/cluster/created/flag.go

```diff
@@ -1,13 +1,24 @@
 package created
 
-import "github.com/spf13/cobra"
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/awscnfm/v12/pkg/env"
+)
 
 type flag struct {
+	TenantCluster string
 }
 
 func (f *flag) Init(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&f.TenantCluster, "tenant-cluster", "c", env.TenantCluster(), "Tenant Cluster ID to use for this particular action.")
```

Comment:
Cluster ID can be given via flag or environment variable. The hustle with the injected kubeconfig does not seem to make much sense after all. So I dropped it again.